gremlin-dotnet/glv/AnonymousTraversal.template

```diff
@@ -27,6 +27,8 @@ using Gremlin.Net.Structure;
 // THIS IS A GENERATED FILE - DO NOT MODIFY THIS FILE DIRECTLY - see pom.xml
 namespace Gremlin.Net.Process.Traversal
 {
+#pragma warning disable 1591
```

Comment:
Looks like you're right. I thought that it would be necessary to also document all function arguments and return values, but it seems to be enough to have a `summary` for each element. This also applies to the `GraphTraversal` class. I'll update the pull request to remove the `pragmas` for both classes.